### ISSUE TITLE
[BITV] 9.2.1.1/5.1 - The button (link) to copy the RSS URL in the "Activity settings" section is not focusable and activatable using the keyboard. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36982]
<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v5-n1

</details>

**Comment by Fenn-CS** [comment]
This is no longer the case. @susnux Can you help me confirm this, so that this can be closed?

**Comment by susnux** [comment]
Resolved in https://github.com/nextcloud/activity/pull/1123
